platform/security/src/test/java/org/codice/alliance/security/claims/videographer/TestVideographerClaimsHandler.java

```diff
@@ -0,0 +1,158 @@
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.security.claims.videographer;
+
+import static java.util.stream.Collectors.joining;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assert.assertThat;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.apache.cxf.rt.security.claims.Claim;
+import org.apache.cxf.rt.security.claims.ClaimCollection;
+import org.apache.cxf.sts.claims.ClaimsParameters;
+import org.apache.cxf.sts.claims.ProcessedClaim;
+import org.apache.cxf.sts.claims.ProcessedClaimCollection;
+import org.apache.wss4j.common.principal.CustomTokenPrincipal;
+import org.codice.alliance.security.principal.videographer.VideographerPrincipal;
+import org.junit.Test;
+
+public class TestVideographerClaimsHandler {
+
+    private static final String CLAIM_URI_1 =
+            "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier";
+
+    private static final String CLAIM_URI_2 =
+            "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress";
+
+    private static final String CLAIM_URI_3 =
+            "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname";
+
+    private static final String CLAIM_VALUE_1 = "Videographer";
+
+    private static final String CLAIM_VALUE_2a = "Videographer@videographer.com";
+
+    private static final String CLAIM_VALUE_2b = "someguy@somesite.com";
+
+    private static final String CLAIM_VALUE_2c = "somedude@cool.com";
+
+    private static final String CLAIM_VALUE_3 = "Videographer";
+
+    private static final String CLAIM1 = CLAIM_URI_1 + "=" + CLAIM_VALUE_1;
+
+    private static final String CLAIM2 = CLAIM_URI_2 + "=" + CLAIM_VALUE_2a;
+
+    private static final String CLAIM3 = CLAIM_URI_3 + "=" + CLAIM_VALUE_3;
+
+    @Test
+    public void testSettingClaimsMapList() throws URISyntaxException {
+        VideographerClaimsHandler claimsHandler = new VideographerClaimsHandler();
+        claimsHandler.setAttributes(Arrays.asList(CLAIM1, CLAIM2, CLAIM3));
+
+        Map<URI, List<String>> claimsMap = claimsHandler.getClaimsMap();
+
+        List<String> value = claimsMap.get(new URI(CLAIM_URI_1));
+        assertThat(value.get(0), is(CLAIM_VALUE_1));
+
+        value = claimsMap.get(new URI(CLAIM_URI_2));
+        assertThat(value.get(0), is(CLAIM_VALUE_2a));
+
+        value = claimsMap.get(new URI(CLAIM_URI_3));
+        assertThat(value.get(0), is(CLAIM_VALUE_3));
+
+        claimsHandler = new VideographerClaimsHandler();
```

Comment:
After examining the VideoClaimsHandler, the original GuestClaimsHandler and GuestClaimsHandlerTest, I realized that these tests are completely worthless. The corresponding tests in GuestClaimsHandlerTest are bogus as well.